### ISSUE TITLE
[Bug]: Minecraft Vanilla Bedrock will not accept players

### ISSUE THREAD

**Issue by wcoleman320** [parkervcp/eggs#1820]
### Panel Version

1.10.1

### Wings Version

1.70

### Service

minecraft

### Modified

No, I did not modify the egg

### Expected Behavior

Fresh install on a linux vm, and i can connect (ubuntu 18.04 server) and it run on windows. Console shows no errors, the server just wont accept connections. Its hows online in the server browser

### Actual Behavior

Client cannot connect to server

### Steps To Reproduce

copy egg-vanilla-bedrock
upload to pterodactyl
create new server and assign 4g of ram 10g storage and allocate port 19132
start server (no errors)
input local ip of daemon and port 19132 into minecraft
server shows online
join server
will not load in

### Install logs

(doesnt work) ls: cannot access '/var/log/pterodactyl/install/*.log': No such file or directory

**Comment by wcoleman320** [comment]
![Screenshot 2022-08-15 185336](https://user-images.githubusercontent.com/7191351/184755970-1b1a6179-9194-4264-af5b-91ab0bb1d529.png)
![Screenshot 2022-08-15 185404](https://user-images.githubusercontent.com/7191351/184755975-1ca84802-7caa-47db-8b0c-dc444cf95f4f.png)
![Screenshot 2022-08-15 185510](https://user-images.githubusercontent.com/7191351/184755979-82cdba41-1d5d-4a72-b7dd-cb61e22e24f7.png)

**Comment by wcoleman320** [comment]
Install log

https://paste-bin.xyz/raw/74961

**Comment by wcoleman320** [comment]
Could possibly be a issue with the dedicated server
https://bugs.mojang.com/projects/BDS/issues/BDS-17499?filter=allissues

**Comment by Ariz-z** [comment]
I am also facing the same issue 
that I install Mineraft bedrock egg but now problem is some user can join and some user is unable to join the server can anyone help whats the issue? <https://github.com/parkervcp/eggs/blob/master/game_eggs/minecraft/bedrock/bedrock/egg-vanilla-bedrock.json> i install this egg

**Comment by xTITUSMAXIMUSX** [comment]
I was able to spin up a bedrock server and connect to it just fine. I actually don't know much about Minecraft but I quickly realized bedrock is not the same as the java version. The bedrock version is **ONLY** for the Windows version. So after my quick google searches I installed the Windows version of the game and I was able to get right in without issue. Maybe this is what everyone is running into. Hope this helps!
